apps/ledger-live-mobile/src/screens/PostOnboarding/PostOnboardingDebugScreen.tsx

```diff
@@ -14,9 +14,11 @@ export default () => {
 
   const handleInitPostOnboardingHub = useCallback(
     (deviceId: DeviceModelId, mock: boolean) =>
-      startPostOnboarding(deviceId, mock, () =>
-        navigation.navigate(NavigatorName.Main),
-      ),
+      startPostOnboarding({
+        deviceModelId: deviceId,
+        mock,
+        fallbackIfNoAction: () => navigation.navigate(NavigatorName.Main),
+      }),
```

Comment:
- this function's signature changed, now passing options in an object for more clarity